lib/utils/mongoclient/src/test/keys/server.key

```diff
@@ -0,0 +1,18 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
@maximilien We use a self-generated CA to self-sign client and server certs and keys. Then we use them to start a test-server and connect with the mongo client. The files are used only as test resources.